crypto/nrf_cc310_mbedcrypto/include/mbedtls_extra/cc_pal_types.h

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2001-2019, Arm Limited and Contributors. All rights reserved.
  *
- * SPDX-License-Identifier: BSD-3-Clause OR Arm’s non-OSI source license
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
is it allowed to remove a license ?

I'm not familiar with such details when it comes to license, but two lines above states that this file is copyrighted by ARM, and so I don't believe we have permission to change the license, if we are not the ones who have copyrighted the file.